src/backends/github/API.js

```diff
@@ -258,6 +263,23 @@ export default class API {
     });
   }
 
+  deleteFile(path, message, options={}) {
+    const branch = options.branch || this.branch;
+    // We need to request the file first to get the SHA
+    return this.request(`${ this.repoURL }/contents/${ path }`)
+    .then(({ sha }) => this.request(
+      `${ this.repoURL }/contents/${ path }`,
+      {
+        method: "DELETE",
```

Comment:
Go with single quotes for this and other quotes in the PR (except React component attributes).